### ISSUE TITLE
Text styling with field inside a table row doesn't work

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#313]
```
Text styling with field inside a table row doesn't work.

See problem with attached zip .



```

Original issue reported on code.google.com by `angelo.z...@gmail.com` on 8 Oct 2013 at 8:15

Attachments:
- [report.zip](https://storage.googleapis.com/google-code-attachments/xdocreport/issue-313/comment-0/report.zip)

**Comment by GoogleCodeExporter** [comment]
```
Problem is fixed for 1.0.3. See Git commit at 
https://code.google.com/p/xdocreport/source/detail?r=99ddbf7c5936634cbd065cd81f5
688dad262a7e2
```

Original comment by `angelo.z...@gmail.com` on 8 Oct 2013 at 9:50

**Comment by GoogleCodeExporter** [comment]
Original comment by `angelo.z...@gmail.com` on 8 Oct 2013 at 9:50
- Changed state: **Fixed**
